tests/suites/0_stateless/20+_others/20_00014_read_small_group.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import time
+
+CURDIR = os.path.dirname(os.path.realpath(__file__))
+sys.path.insert(0, os.path.join(CURDIR, "../../../helpers"))
+
+from client import client
+
+log = None
+
+client1 = client(name="client1>", log=log)
+
+sqls = """
+set storage_io_min_bytes_for_seek=0;
+SELECT * from @data/ontime_200.parquet;
+"""
+t1 = time.time_ns()
+client1.run(sqls)
+t2 = time.time_ns()
+
+
+sqls = """
+set storage_io_min_bytes_for_seek=43;
+SELECT * from @data/ontime_200.parquet;
+"""
+t3 = time.time_ns()
+client1.run(sqls)
+t4 = time.time_ns()
+
+assert (t4-t3)/(t2-t1) < 0.8 
```

Comment:
Unit test is ok, we can not do logic test for the performance. We should do in the unit test: check the merge logic is impl as expected, avoid someone change the codes in the future break it.